scripts/install

```diff
@@ -29,53 +52,70 @@ fi
 sudo apt-get install php5-cli -y -qq
 
 # Copy files to the right place
-app_home_path=/home/yunohost.app/shaarli
-final_path=/var/www/shaarli
+app_home_path=/home/yunohost.app/$app
+final_path=/var/www/$app
 
 sudo mkdir -p $final_path
 # Download and extract in /var/www
 extract_shaarli
 
-sudo mkdir -p $app_home_path/data
-sudo mkdir -p $app_home_path/cache
-sudo mkdir -p $app_home_path/pagecache
-sudo mkdir -p $app_home_path/tmp
-sudo mkdir -p $app_home_path/conf
-sudo chown -R www-data: $app_home_path
-
+# generate the salt
 salt=$(dd if=/dev/urandom bs=1 count=200 2> /dev/null | tr -c -d 'A-Za-z0-9' | sed -n 's/\(.\{24\}\).*/\1/p')
+
+# generate the hash with the password
+password_hash=$(php ../conf/gen_hash.php $password $admin $salt)
+
+# set the proper value for substitution
 if [ $privatelinkbydefault = "No" ]; then
```

Comment:
You're right, no real to skip this step as we have to prompt the user for a password anyway.
Let's keep this simple.